pyanaconda/modules/storage/devicetree/devicetree.py

```diff
@@ -131,3 +136,26 @@ def _get_device_attrs(self, device):
             attrs[name] = str(value)
 
         return attrs
+
+    def get_actions(self):
+        """Get the device actions.
+
+        :return: a list of DeviceActionData
+        """
+        return [
+            self._get_action_data(action)
+            for action in self.storage.devicetree.actions.find()
+        ]
```

Comment:
The `for` on the separate line is hard to read and even more with those brackets. Could you please rewrite this part.

Reply:
Fixed.